### PR TITLE
Make grid size to integer for GEMM example

### PR DESCRIPTION
**Pull request by yuyu2172**
The code broke after https://github.com/cupy/cupy/pull/108

### CODE REVIEW

File: examples/gemm/sgemm.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def sgemm(A, B,
     code = read_code(sgemm_file, params=config)
     kern = load_kernel('sgemm', code)
 
-    grid = (math.ceil(m / blk_m), math.ceil(n / blk_n), 1)
+    grid = (int(math.ceil(m / blk_m)), int(math.ceil(n / blk_n)), 1)
```
Comment (okuta) [inline examples/gemm/sgemm.py]: How about using `//`?

Comment (yuyu2172) [inline examples/gemm/sgemm.py]: Wouldn't it change the behavior?
For instance, when `m=3` and `blk_m=2`, `math.ceil(m / blk_m) == 2` and `m // blk_m == 1`.

Comment (okuta) [inline examples/gemm/sgemm.py]: sorry, I misunderstood.

**Comment by okuta** [comment]
OK

**Pull request merged**